doc/source/structures/misc/string.rst

```diff
@@ -127,6 +169,9 @@ Structure
         * - :meth:`INSERT(index, string)`
           - :struct:`String`
           - Returns a new string with the given string inserted at the given index into this string
+        * - :attr:`ITERATOR`
```

Comment:
In all of our other structures, we don't duplicate suffixes from inherited structure types.  But this table doesn't direct the user to go look for `Enumerator`.  I think I will add that reference and delete the `ITERATOR` documentation in this file if that's OK with you.  It also fixes an error since Sphinx is complaining that it's defined twice.